src/view/event-item-view.js

```diff
@@ -0,0 +1,59 @@
+import {createElement} from '../render.js';
```

Comment:
```suggestion
import { createElement } from '../render.js';
```

либо везде с пробелами, либо без!